docs/deployment/README.md

```diff
@@ -39,18 +39,34 @@ The process involves:
 
 #### Install Terraform
 
-If you are using a Mac and [Homebrew](https://brew.sh), installing Terraform is
+It is recommended to keep a consistent version of Terraform as you deploy. Once your state files are migrated to a higher version, they are not always backwards compatible so integrators should pin their Terraform version. This is easily accomplished using the Terraform Version Manager [tfenv](https://github.com/tfutils/tfenv).
```

Comment:
We should probably combine the line 77 with this paragraph? Or update the note on line 77 to match the recommendation here of keeping your Terraform version consistent across all machines using tfenv

https://github.com/nasa/cumulus/pull/1378/files#diff-d62df7ce64b0930247dc52b3a8251832R77

Reply:
Updated.